tools/src/GitHub.ts

```diff
@@ -54,6 +54,40 @@ export async function getPullRequestAsync(
   return data;
 }
 
+/**
+ * Returns the url of the PR that closed an issue.
+ */
+export async function getIssueCloserPrUrlAsync(issueNumber: number): Promise<string> {
```

Comment:
Unfortunately this info is only available through the graphql api

Reply:
fancy though